### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zaknafean**
Added some undocumented features, so others don't have to go through the source to find them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ SyndiBox is meant to be an easy and stress-free way of implementing dialog into
  4. Fill the properties in the Inspector with your dialog, auto advance, font, text voice, color, and text speed. You can use what is filled in the image below as an example.![Fill the Inspector properties.](https://imgur.com/0POjPSz.png)
  5. Press the `Play Scene` button (or `F6` on your keyboard) and watch it print!![There it is~](https://imgur.com/Fiigoty.png)
 
+## Script Guide
+Syndibox exposes some functions for use within GDScript that allow for dynamic control of the text and effects.   
+
+start(string: dialog, int: start_position) - Resets, shows, and starts the dialog box, displaying the given string. Passing a string is functionally the same as if you entered the string as Dialog in the inspector. dialog defaults to ''. Start position indicates where to beging displaying the text. start_position defaults to 0.
```
Comment (I-Have-No-Idea-What-IAmDoing) [inline README.md]: It should be noted that leaving the dialog value blank just reprints the current dialog set

Comment (zaknafean) [reply README.md]: Ah another good thing to be documented! I'll edit this.

**Review by Sudospective — COMMENTED** [review]
@I-Have-No-Idea-What-IAmDoing You might want to look over this again before I approve, I recall you added a tagging system and I think it would be a good idea if you added documentation for that if you have the time. No pressure.

**Comment by zaknafean** [comment]
The character switching tags and code are definitely something that will require a bit more time and review then I think I can give, as its not fully obvious to me what its doing in the source AND seems to assume a specific folder structure. This readme update is more or less just explaining functionality that I had to go through source to find, nothing advanced.

**Comment by zaknafean** [comment]
Going to add some of the new merged tags in now (signaling etc), now that I've grabbed the latest code.

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
The only thing that is really missing is manual pausing/hiding which is controlled by a variable right now (should i make a function for it? make it a export?) and tag creation would be useful. 

Tags start with [ and end with ], you need some sort of content inside of them and it it can be any character including white space. If the tag is not valid then it get printed so you can use [] in the dialog without it being deleted. you can play around with the regex here: https://regexr.com/5eaih if you need it

**Comment by zaknafean** [comment]
Variable is fine as long as we documented. The variable with the get/set is more the Godot design paradigm anyway. I'll add it to the script section after I try it!

Is the tag creation like for custom tags? So stuff you'd add to the plugin code directly?

**Review by Sudospective — APPROVED** [review]
Thank you very much!

**Comment by I-Have-No-Idea-What-IAmDoing** [comment]
Tag creation is for custom tag, that go in the custom.gd script

**Pull request merged**